Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubelet/util.go

```diff
@@ -73,3 +93,31 @@ func allowHostNetwork(pod *api.Pod) (bool, error) {
 	}
 	return false, nil
 }
+
+// Determined whether the specified pod is allowed to use host networking
+func allowHostPID(pod *api.Pod) (bool, error) {
+	podSource, err := getPodSource(pod)
+	if err != nil {
+		return false, err
+	}
+	for _, source := range capabilities.Get().PrivilegedSources.HostPIDSources {
+		if source == podSource {
+			return true, nil
+		}
+	}
+	return false, nil
+}
+
+// Determined whether the specified pod is allowed to use host ipc
```

Comment:
Same typo.  We probably don't need to worry about fixing it unless there's something else substantial to fix about this.